app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SessionsController < Devise::SessionsController
+  def new
+    super
+  end
+
+  def create
+    if(current_company == nil)
+      flash[:danger] = 'Invalid Company!'
+      return redirect_to new_employee_session_path
+    end
+    employee = Employee.find_by(email: params[:employee][:email], company_id: current_company.id)
```

Comment:
what if params[:employee] is blank?

Reply:
then employee will be nil and invalid email or password will appear

Reply:
if params[:employee] is blank; params[:employee][:email] will crash
undefined [] for nil class error

Reply:
checked Invalid error will appear won't crash